src/api/store/testimonial.py

```diff
@@ -74,7 +74,7 @@ def create_testimonial(self, context: Context, args) -> Tuple[dict, MassEnergize
       action = args.pop('action', None)
       vendor = args.pop('vendor', None)
       community = args.pop('community', None)
-      user_email = context.user_email
+      user_email = args.pop("user_email") or context.user_email
```

Comment:
For security reasons, we would want the backend to be the sole source of truth for user credentials (email and ID) for routes like this.

Reply:
We can use the context to get the user instead of passing an email or id from the frontend